onmt/modules/copy_generator.py

```diff
@@ -91,72 +90,73 @@ def forward(self, hidden, attn, src_map):
         # Original probabilities.
         logits = self.linear(hidden)
         logits[:, self.tgt_dict.stoi[inputters.PAD_WORD]] = -float('inf')
-        prob = self.softmax(logits)
+        prob = F.softmax(logits, 1)
 
         # Probability of copying p(z=1) batch.
-        p_copy = self.sigmoid(self.linear_copy(hidden))
+        p_copy = F.sigmoid(self.linear_copy(hidden))
```

Comment:
https://pytorch.org/docs/stable/_modules/torch/nn/functional.html
Not sure this is the best way to change this since it's deprecated.